ucx.spec

```diff
@@ -106,19 +119,19 @@ Provides header files and examples for developing with UCX.
            --disable-assertions \
            --disable-params-check \
            --without-java \
-	   --without-go \
-           %_enable_arg cma cma \
-           %_with_arg cuda cuda \
-           %_with_arg gdrcopy gdrcopy \
-           %_with_arg ib verbs \
-           %_with_arg ib_cm cm \
-           %_with_arg knem knem \
-           %_with_arg rdmacm rdmacm \
-           %_with_arg rocm rocm \
-           %_with_arg xpmem xpmem \
+           --without-go \
+          %_enable_arg cma cma \
+          %_with_arg cuda cuda \
+          %_with_arg gdrcopy gdrcopy \
+          %_with_arg ib verbs \
+          %_with_arg ib_cm cm \
+          %_with_arg knem knem \
+          %_with_arg rdmacm rdmacm \
+          %_with_arg rocm rocm \
+          %_with_arg xpmem xpmem \
```

Comment:
Inadvertent removal of leading whitespace?  These lines should line up with the lines above them.